web/src/app/schedules/temp-sched/TempSchedShiftsList.tsx

```diff
@@ -121,7 +120,7 @@ export default function TempSchedShiftsList({
 
           return {
             scrollIntoView: true,
-            id: s.start + s.userID,
+            id: s.userID + index.toString(),
```

Comment:
```suggestion
            id: s.start + s.userID + index.toString(),
```
We'll want to add the start time back in otherwise we might get multiple list items with the same id.